src/utils/getAWSRegion.js

```diff
@@ -0,0 +1,6 @@
+const getAWSRegion = () => {
+  const region = process.env.AWS_REGION || process.env.DEFAULT_REGION || 'eu-west-1';
```

Comment:
can you remove the default region hardcoding to 'eu-west-1' in here, make sure region is defined when in development environment, and raise an error in here if region is not defined?